### DISCUSSION TITLE
Оптимизация для АТВ, Сделать туннелирование для выбранных приложений

### DISCUSSION THREAD

**Discussion by Killergy** [dovecoteescapee/ByeDPIAndroid#113] [Ideas]
Здравствуйте, возможно ли сделать это приложение для атв, и сделать туннелирование отдельный приложений. К примеру так же хотелось бы иметь возможность чтобы в Smartube, настройки, общие интернет цензура приложение можно было запускать через прокси этого приложения... Такое возможно?.

**Comment by Killergy** [comment]
Очень хотелось бы это видеть в будущем обновлении. Так ежели приложение может стать полной заменой PowerTunnel , которые не работает адекватно по причине установки сертификатов

**Comment by dovecoteescapee** [comment]
Split tunneling в планах. О проблемах на Android TV я получаю много сообщений, но у меня нет соответствующего девайса, чтобы протестировать

**Reply by Killergy** [reply]
В ВПН смартуб загрузился на атв. Но вот в прокси не умеет как повертунель работать со смартубом... Проблема ВПН в том что он начинает все приложения через себя пропускать.. спасает приложение для рутованных устройств дпи туннель, тот умеет для выбранных приложений мотать трафик.

**Reply by dovecoteescapee** [reply]
А какая проблема возникает с прокси?

**Reply by Killergy** [reply]
С прокси я сам лоханулся, я ставил в хттп а надо в сокс. Единственное, что тест показывает что к Ютубу есть подключение, а вот второй тест к Гуглу пишет java.net.socketTimeoutException:SSL Handshake timed out
Когда ВПН в Смартубе в режиме прокси напротив выдает что оба теста проходить как в Ютубе так и с Гуглом

**Reply by Killergy** [reply]
> С прокси я сам лоханулся, я ставил в хттп а надо в сокс. Единственное, что тест показывает что к Ютубу есть подключение, а вот второй тест к Гуглу пишет java.net.socketTimeoutException:SSL Handshake timed out Когда ВПН в Смартубе в режиме прокси напротив выдает что оба теста проходят как в Ютубе так и с Гуглом
